README.md

```diff
@@ -260,6 +266,12 @@ If you want to be much less precise, you can use `=`.
 3 = 3.14! //true
 ```
 
+You can also be a lot less precise by using `||`
+
+```java
+3 || 2! //true
+```
+
```

Comment:
I'd drop this one, but keep the other two. Then I'd be happy to merge :)

Reply:
Ok